### PR TITLE
Put the count down widget in place

### PR DESCRIPTION
**Pull request by starbelly**
As is tradition!

Screen Shot 2021-09-17 at 4 55 45 PM

Don't need to merge this until Friday 11:59pm UTC

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Hm... in previous editions, the countdown was the time until the end of the event (48 hours - like some kind of pressure), but I don't mind changing that to "start of event".

Also, is it difficult to put the symbol instead of text (I'm guessing it is, and that's why you went with the text version)? If not, I'd prefer it, but this shouldn't keep us from merging.

👍

**Comment by starbelly** [comment]
@paulo-ferraz-oliveira that's what the PR is for, it won't be merged until Friday at 11:59 pm UTC :)

**Comment by starbelly** [comment]
> Is it difficult to put the symbol instead of text (I'm guessing it is, and that's why you went with the text version)? If not, I'd prefer it, but this shouldn't keep us from merging.
> 
> 👍

I could try to resize it maybe. I'll give it a go later, timebox it to 15 minutes.

File: vendor/countdown_element.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var Countdown = new Countdown({
-  year     : 2019,
+  year     : 2021,
   month    : 9,
-  day      : 23,
+  day      : 19,
```
Comment (paulo-ferraz-oliveira) [inline vendor/countdown_element.js:4]:
```suggestion
  day      : 20,
```

Comment (starbelly) [reply vendor/countdown_element.js]: Resolving this as obsolete, noticed before my last push.

**Review by pablocostass — APPROVED** [review]

**Pull request merged**